monai/transforms/utility/array.py

```diff
@@ -111,16 +112,23 @@ class AsChannelFirst(Transform):
         channel_dim: which dimension of input image is the channel, default is the last dimension.
     """
 
+    backend = [TransformBackends.TORCH, TransformBackends.NUMPY]
+
     def __init__(self, channel_dim: int = -1) -> None:
         if not (isinstance(channel_dim, int) and channel_dim >= -1):
             raise AssertionError("invalid channel dimension.")
         self.channel_dim = channel_dim
 
-    def __call__(self, img: np.ndarray) -> np.ndarray:
+    def __call__(self, img: NdarrayOrTensor) -> NdarrayOrTensor:
         """
         Apply the transform to `img`.
         """
-        return np.moveaxis(img, self.channel_dim, 0)
+        # old versions of pytorch don't have moveaxis
```

Comment:
May I know which version start to support it? We only need to support PyTorch >= 1.5.
Can we use `torch.permute()` or `torch.transpose()` if no `moveaxis`?

Thanks.

Reply:
Sure, updated to use `torch.permute`.